docs/articles/toc.yml

```diff
@@ -1,66 +1,76 @@
 - name: Introduction
   href: intro.md
-- name: Getting started
+- name: The Basics
   items:
-  - name: Bot application
-    href: getting_started/bot_application.md
-  - name: Basic bot
-    href: getting_started/basic_bot.md
-  - name: Alternate WebSocket clients
-    href: getting_started/alternate_ws.md
-  - name: Event reference
-    href: getting_started/event_reference.md
-  - name: Notes about using Mono
-    href: getting_started/mono.md
+  - name: Creating a Bot Application
+    href: basics/bot_application.md
+  - name: Writing a Basic Bot
+    href: basics/basic_bot.md
+- name: Beyond Basics
+  items:
+  - name: Understanding TAP Events
+    href: beyond_basics/event_reference.md
+  - name: Alternate WebSocket Clients
+    href: beyond_basics/alternate_ws.md
+  - name: Using the Mono Runtime
+    href: beyond_basics/mono.md
+  - name: Sharding
+    href: beyond_basics/sharding.md
 - name: Commands
   items:
   - name: Introduction
     href: commands/intro.md
-  - name: Argument converters
+  - name: Argument Converters
     href: commands/argument_converters.md
-  - name: Dependency injection
-    href: commands/dependency_injection.md
-  - name: Module lifespans
+  - name: Module Lifespans
     href: commands/module_lifespans.md
-  - name: Custom command handlers
+  - name: Dependency Injection
+    href: commands/dependency_injection.md
+  - name: Custom Command Handler
     href: commands/custom_handler.md
+  - name: Command Attributes
+    href: commands/command_attributes.md
 - name: Interactivity
   items:
   - name: Introduction
     href: interactivity/intro.md
-- name: Voice
+- name: Audio
   items:
-  - name: Setup
-    href: voice/setup.md
-  - name: Sending
-    href: voice/send.md
-  - name: Receiving
-    href: voice/receive.md
-- name: Lavalink
-  items:
-  - name: Introduction
-    href: lavalink/intro.md
+  - name: VoiceNext
+    items:
+    - name: Introduction
+      href: audio/voicenext/introduction.md
+    - name: Sending
+      href: audio/voicenext/send.md
+    - name: Receiving
+      href: audio/voicenext/receive.md
+  - name: Lavalink
+    items:
+    - name: Introduction
+      href: audio/lavalink/intro.md
+    - name: Implementation
+      href: audio/lavalink/implementation.md
+    - name: Music Commands
+      href: audio/lavalink/music_commands.md
 - name: Hosting
   items:
-  - name: Running your bot 24/7
+  - name: Running Your Bot 24/7
     href: hosting/options.md
   - name: Hosting on Raspbery Pi
     href: hosting/raspberry_pi.md
-- name: Migrating after major updates
+- name: Migration
```

Comment:
"Migration" alone is vague and unindicative of what this section of the docs is dedicated to. The previous title was fine.

Reply:
I was planning on improving this later. The previous title was much too long, so I shortened it to *Migration* for the time being.

Reply:
Previous title was fine, it was clear about what the guides in the section are for.